Robert Stoelhorst/week_02/day03/index.css

```diff
@@ -0,0 +1,5 @@
+/* .cat {
+  position: absolute;
+  left: 0px;
+  top: 350px;
+} */
```

Comment:
The default name for a CSS file is normally something like style.css. Index is usually the default name for the html homepage.